docs/automated-testing/unit-testing/mocking.md

```diff
@@ -92,16 +92,128 @@ since it is held to the same standard with full unit tests. Especially if writin
 developers can use, providing a fake in the product code means those developers no longer need to write their own mock
 implementations, further increasing re-usability of code.
 
-### Upsides
+### Upsides to Fakes
 
 - No framework needed, is just like any other implementation.
 - Encourages testable design.
 - Code can be "promoted" to product code, so it is not wasted effort.
 
-### Downsides
+### Downsides to Fakes
 
 - Takes more time to implement.
 
+## Best Practices
+
+To keep your mocking efficient, consider these best practices to make your code testable, save time and make your
+test assertions more meaningful.
+
+### Dependency Injection
+
+If you don’t keep testability in mind from the beginning, once you start writing your tests, you might realize you have
+to do a time-intensive refactor to make the code unit testable. A common problem that leads to non-testable code is not
+using dependency injection. Use dependency injection so that a mock can easily be injected into your SUT during a unit test.
```

Comment:
```suggestion
using dependency injection. Use dependency injection so that a mock can easily be injected into your Subject Under Test (SUT) during a unit test.
```